src/unix/darwin.c

```diff
@@ -333,3 +333,90 @@ void uv_free_interface_addresses(uv_interface_address_t* addresses,
 
   uv__free(addresses);
 }
+
+int uv__enable_largesocket(uv_stream_t *stream) {
+  int rmem;
+  int wmem;
+  int ret;
+  int rs;
+  int ws;
+  int rr;
+  int rw;
+  char *buf, *buf2, *pattern, *token, *token2;
+  struct stat statistics;
+  int fd = open("/etc/sysctl.conf", O_RDONLY);
```

Comment:
I had thought about this, but did not find net.inet* probes / properties under the head CTL_NET in sys/socket.h. 

Excerpts from man sysctl:
 
```
The top level names are defined with a CTL_ prefix in <sys/sysctl.h>, and are as follows.  
The next and subsequent levels down are found in the include files listed here, 
and described in separate sections below.

           Name              Next level names          Description
           CTL_DEBUG         sys/sysctl.h              Debugging
           CTL_VFS           sys/mount.h               File system
           CTL_HW            sys/sysctl.h              Generic CPU, I/O
           CTL_KERN          sys/sysctl.h              High kernel limits
           CTL_MACHDEP       sys/sysctl.h              Machine dependent
           CTL_NET           sys/socket.h              Networking
           CTL_USER          sys/sysctl.h              User-level
           CTL_VM            sys/resources.h           Virtual memory (struct loadavg)

```
`bash$ find /usr/include | xargs grep -E "sendspace|recvspace" 2> /dev/null`

yielded nothing.